src/licensedcode/models.py

```diff
@@ -655,8 +662,22 @@ def validate(licenses, verbose=False, no_dupe_urls=False):
                 if not len(all_licenses) == len(set(all_licenses)):
                     warn('Some duplicated URLs')
 
-            # local text consistency
             text = lic.text
+            if thorough:
+                # local text consistency
+                data = {"text": text}
+                # We are testing whether we can dump as yaml and load from yaml
+                # without failing (i.e. whether the text is yaml safe)
+                # Using saneyaml
+                try:
+                    yaml_string = saneyaml_dump(data, indent=4)
```

Comment:
Why do you use indent 4 as opposed to the default?

Reply:
removing this, unintended.